chart/kubenab/templates/deployment.yaml

```diff
@@ -1,12 +1,9 @@
 {{- if .Values.docker.registryUrl }}
-{{- if .Capabilities.APIVersions.Has "apps/v1" }}
 apiVersion: apps/v1
-{{- else }}
-apiVersion: extensions/v1beta1
-{{- end }}
```

Comment:
This was added to keep a Backward Compatibility.
So why do you wanna remove this?

Reply:
Added back in.